phing/tasks/deploy.xml

```diff
@@ -156,4 +156,12 @@
       </fileset>
     </delete>
   </target>
+
+  <target name="deploy:update" description="Update current database to reflect the state of the Drupal file system; uses local drush alias.">
```

Comment:
This is actually using "self" rather than the local alias, which is drush.aliases.local. Can you also add a comment explaining the need for the vcs value?